### ISSUE TITLE
[Feature request] Support HTML in Knowledgebase

### ISSUE THREAD

**Issue by PeopleInside** [uvdesk/community-skeleton#331]
**Description**  

Actually seems there is no way to insert HTML elements on the Knowledge base

Add support for HTML in the editor or articles cannot have HTML element.

**Comment by vaishaliwebkul** [comment]
@PeopleInside 
 I am able to add HTML code inside KnowledgeBase(Articles). their is rich text editor provided in the article which support the "source code" feature using which we can add our code.

![Screenshot_71](https://user-images.githubusercontent.com/43210406/96346827-8159b200-10bb-11eb-9a27-7ddb0c4c0bfd.png)
![Screenshot_72](https://user-images.githubusercontent.com/43210406/96346835-8a4a8380-10bb-11eb-95dc-74fcb9fe19c5.png)

**Comment by PeopleInside** [comment]
I tried to put a PayPal button, code taken from PayPal and the code is reset, not showed in the public article of the KnowledgeBase.
Let me try again now.
Tested with the donate button taken from PayPal.

Unable to insert the PayPal button so there are some issue.
Please try to include a PayPal donation button. Open PayPal, settings, selling tools. Donate button and take the code.

**Comment by vaishaliwebkul** [comment]
@PeopleInside 
 Okay, got it, I will update you on this soon.

Thanks.

**Comment by PeopleInside** [comment]
Maybe this issue should be transferred https://github.com/uvdesk/support-center-bundle/issues
Can be related to the following bug: https://github.com/uvdesk/support-center-bundle/issues/112
